packages/circuit-ui/components/SideNavigation/components/DesktopNavigation/DesktopNavigation.tsx

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable jsx-a11y/no-redundant-roles */
+
+import { css } from '@emotion/core';
+import { Theme } from '@sumup/design-tokens';
+
+import styled, { StyleProps } from '../../../../styles/styled';
+import { shadow, hideScrollbar } from '../../../../styles/style-mixins';
+import { useFocusList } from '../../../../hooks/useFocusList';
+import { TOP_NAVIGATION_HEIGHT } from '../../../TopNavigation/TopNavigation';
+import Headline from '../../../Headline';
+import { PrimaryLinkProps } from '../../types';
+import { SecondaryLinks } from '../SecondaryLinks';
+import { PrimaryLink } from '../PrimaryLink';
+
+export interface DesktopNavigationProps {
+  /**
+   * TODO: Add description
+   */
+  primaryLinks: PrimaryLinkProps[];
+  /**
+   * TODO: Add description
+   */
+  primaryNavigationLabel: string;
+  /**
+   * TODO: Add description
+   */
+  secondaryNavigationLabel: string;
+}
+
+const PRIMARY_NAVIGATION_WIDTH = '48px';
+
+const wrapperStyles = ({ theme }: StyleProps) => css`
+  ${theme.mq.untilGiga} {
+    display: none;
+  }
+`;
+
+const Wrapper = styled.div(wrapperStyles);
+
+const primaryWrapperStyles = ({ theme }: StyleProps) => css`
+  position: fixed;
+  z-index: ${theme.zIndex.navigation + 2};
```

Comment:
Is it a breaking change to require a `@sumup/design-tokens` bump to have the `navigation` value available?

Reply:
As we discussed offline, no. It only requires users to upgrade `@sumup/design-tokens` alongside `@sumup/circuit-ui` which is a reasonable requirement.